virtool/faker.py

```diff
@@ -0,0 +1,10 @@
+from faker import Faker
+from faker.providers import misc
+
+class VirtoolFaker(Faker):
+    fake = Faker()
+    Faker.seed(0)
+    fake.add_provider(misc)
+
+    def get_mongo_id(self):
+        return self.fake.password(length=8, special_chars=False)
```

Comment:
Call it `FakerWrapper`.

Should be in `virtool/faker/wrapper.py`. We will need the faker directory for other module files.